<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XyzHttp</class>
 <widget class="QMainWindow" name="XyzHttp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XyzHttp</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="vlayoutMain">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupReq">
      <property name="title">
       <string>Request</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="hLayoutUrl">
         <item>
          <widget class="QLabel" name="labUrl">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editUrl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayoutParam">
         <item>
          <widget class="QLabel" name="labMethod">
           <property name="text">
            <string>METHOD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboMethod">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>POST</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GET</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DELETE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PUT</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labContType">
           <property name="text">
            <string>Content-Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboContType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>application/json</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/x-www-form-urlencoded</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/xml</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/xhtml+xml</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/octet-stream</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/pdf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>application/msword</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>multipart/form-data </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text/html</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text/plain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text/xml</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>image/gif</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>image/jpeg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>image/png</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labCharset">
           <property name="text">
            <string>Charset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboCharset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="hspacerC">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupParam">
      <property name="title">
       <string>Param</string>
      </property>
      <layout class="QHBoxLayout" name="hLayoutParam">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageJson">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="txtJson">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageTable">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableView" name="tableParam">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFile">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="hlayoutFile">
             <item>
              <widget class="QLabel" name="labFile">
               <property name="text">
                <string>File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editFile">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSelectFile">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hspacerFile">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>208</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="vspacerP">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>162</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupResp">
      <property name="title">
       <string>Response</string>
      </property>
      <layout class="QHBoxLayout" name="hLayoutResp">
       <item>
        <widget class="QTabWidget" name="tabResp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabHeader">
          <attribute name="title">
           <string>Header</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPlainTextEdit" name="txtHeader">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabBody">
          <attribute name="title">
           <string>Body</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPlainTextEdit" name="txtBody">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabBeauty">
          <attribute name="title">
           <string>Beauty</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPlainTextEdit" name="txtBeauty">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
